3_javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,143 @@
+const freeCampCodeUser = "@sevda-89";
+console.log(freeCampCodeUser);
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+for (let i = 0; i < names.length; i++) {
+  if (names[i] === nameToRemove) {
+    names.splice(i, 1);
+    i--;
+  }
+}
+
+console.log(names);
+// ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+//When will we be there??
+
+function calculateTravelTime(travelInformation) {
+  const speed = travelInformation.speed;
+  const distance = travelInformation.destinationDistance;
+
+  // Calculate the time in hours and minutes
+  const hours = Math.floor(distance / speed);
+  const minutes = Math.floor((distance % speed) / (speed / 60));
+
+  // Format the time
+  const formattedTime = `${hours} hours and ${minutes} minutes`;
+
+  return formattedTime;
+}
+
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+const travelTime = calculateTravelTime(travelInformation);
+console.log(`I will arrive ${travelTime} later`);
+
```

Comment:
🎊 Your logic to determine the travel time is straightforward and easy to understand.
Good use of the Math.floor function to ensure that the values for hours and minutes are rounded down appropriately.
❗ Areas of Improvement:

**Code Redundancy:**
You've accessed properties using `travelInformation.speed` and `travelInformation.destinationDistance`. Using object destructuring would be more concise.

**Variable Naming**:
Consider renaming `distance` to `totalDistance` to make it clearer.
**Commenting**:
A brief comment about why you used the particular logic for minutes would be helpful.